lib/generateMarkdown.js

```diff
@@ -373,29 +373,28 @@ function getPropertySummary(property, knownTypes, autoLink) {
  * it will fall back to trying to get the values from the anyOf object.
  * @param  {object} schema The schema object that may be of an enum type.
  * @param  {string} type The name of the object type for the enum values (e.g. string, integer, etc..)
+ * @param  {integer} depth How deep the bullet points for enum values should be.  Maximum is 2.
  * @return {string} A string that enumerates all the possible enum values for this schema object.
  */
-function getEnumString(schema, type) {
+function getEnumString(schema, type, depth) {
     var propertyEnum = schema['enum'];
     if (!defined(propertyEnum)) {
         // It's possible that the enum value is defined using the anyOf construct instead.
-        return getAnyOfEnumString(schema, type);
+        return getAnyOfEnumString(schema, type, depth);
     }
 
     var propertyEnumNames = schema['gltf_enumNames'];
 
     var allowedValues = '';
+    var prefix = '\n' + ('      '.substring(0, depth * 3)) + '* ';
```

Comment:
Should this be a one-liner that comes out of style?  It seems like `style` should be the unified place where all markup logic is...so, something like `style.getBulletPoint(element, depth)`, and that would both figure out the `prefix` and do the concatenation that you're doing later in `397`.